Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,169 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+    }
+    // Print particle's shape factors
+    amrex::Print()<<"Particle Shape Factor:| " << WarpX::nox <<"\n";
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n";
+    // Print solver's type: Yee, CKC, ECT
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::Yee){
+      amrex::Print()<<"Maxwell Solver:       | yee\n";
+      }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::CKC){
+      amrex::Print()<<"Maxwell Solver:        | ckc" <<"\n";
+    }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT){
+      amrex::Print()<<"Maxwell Solver:        | ect" <<"\n";
+    }
+  #ifdef WARPX_USE_PSATD
+    // Print PSATD solver's configuration
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+      amrex::Print()<<"Maxwell Solver:       | psatd\n";
+      }
+    if ((m_v_galilean[0]!=0) or (m_v_galilean[1]!=0) or (m_v_galilean[2]!=0)) {
+      amrex::Print()<<"                      | - Galilean \n"<<
+      "                      |  - v_galilean=("<<m_v_galilean[0]<<","<<m_v_galilean[1]<<","<<m_v_galilean[2]<<")\n";
+      }
+    if (WarpX::update_with_rho==1) {
+      amrex::Print()<<"                      | - update with rho\n";
+      }
+    if (WarpX::do_dive_cleaning==1) {
+      amrex::Print()<<"                      | - div(E) cleaning\n";
+      }
+    if (WarpX::do_divb_cleaning==1) {
+      amrex::Print()<<"                      | - div(B) cleaning\n";
+      }
+    if (do_multi_J == 1){
+        amrex::Print()<<"                      | - multi-J deposition\n";
+        amrex::Print()<<"                      |   - J-deposition number="<<WarpX::do_multi_J_n_depositions<<"\n";
+    }
+    if (fft_do_time_averaging == 1){
+      amrex::Print()<<"                      | - time-averaged \n";
+    }
+  #endif // WARPX_USE_PSATD
+
+  if (do_nodal==1){
+    amrex::Print()<<"                      | - nodal mode" <<"\n";
+  }
+  #ifdef WARPX_USE_PSATD
+    if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::EnergyConserving) ){
+      amrex::Print()<<"                      | - staggered mode " <<"\n";
+    }
+    else if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::MomentumConserving) ){
+      amrex::Print()<<"                      | - hybrid mode\n";
+      amrex::Print()<<"                      |   - field_centering_nox="<<WarpX::field_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noz<<"\n";
+      amrex::Print()<<"                      |   - current_centering_nox="<<WarpX::current_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noz<<"\n";
```

Comment:
I would put a space around the `=` sign, to make it more readable:
```suggestion
      amrex::Print()<<"                      |   - field_centering_nox = "<<WarpX::field_centering_nox<<"\n";
      amrex::Print()<<"                      |   - field_centering_noy = "<<WarpX::field_centering_noy<<"\n";
      amrex::Print()<<"                      |   - field_centering_noy = "<<WarpX::field_centering_noz<<"\n";
      amrex::Print()<<"                      |   - current_centering_nox = "<<WarpX::current_centering_nox<<"\n";
      amrex::Print()<<"                      |   - current_centering_noy = "<<WarpX::current_centering_noy<<"\n";
      amrex::Print()<<"                      |   - current_centering_noy = "<<WarpX::current_centering_noz<<"\n";
```